src/controller/handler/connections.ts

```diff
@@ -83,18 +83,15 @@ export class Connections implements IAgentResponseHandler {
 
             Logger.info(`requesting agent to accept connection invite ${req.url}`);
             const res = await this.http.requestWithRetry(req);
-            return res;
+            return res.data;
```

Comment:
I was getting an error converting circular JSON when trying to return the entire res object, instead just returning data